src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
```

Comment:
Is it ok to remove the trailing whitespace?

Reply:
Yeah that's fine as long as CI is happy.